lib/liquid/block_body.rb

```diff
@@ -131,6 +131,11 @@ def blank?
       @blank
     end
 
+    def remove_blank_strings
```

Comment:
Might be worth commenting some of the details about `@blank` here to make it clear when/why this should be called.